start.js

```diff
@@ -0,0 +1,61 @@
+const http = require("http");
+const soldierRoute = require("./routs/soldiers.js");
+const dutiesRoute = require("./routs/duties");
+const jbRoute = require("./routs/justiceBoard");
+
+function runServer() {
+  let server = http.createServer(function (req, res) {
+    let {
+      method,
+      url
+    } = req;
+
+    let body = [];
+    req.on('data', (chunk) => {
+      body.push(chunk);
+    }).on('end', () => {
+      body = Buffer.concat(body).toString();
+      if (body != "") {
+        body = JSON.parse(body);
+      }
+      if (/soldiers.*/.test(url.split("/")[1]) && url.split("/").length <= 3) {
```

Comment:
this is way to big. split to multiple functions. use switch case to distinguish cases

Reply:
decouple your logic, no single method should be this big